auth/utils/index.ts

```diff
@@ -24,6 +24,6 @@
  */
 
 export * from './helpers';
-export * from './read-credentials-file';
+export * from './file-reading-helpers';
```

Comment:
I changed the name of this file to be more accurate. Even though this file is importable by users, it is not part of our "official" API so I don't consider it a breaking change. We wouldn't expect any users to be using this file, it only contains internal utilities.